backend/Services/SightingService.cs

```diff
@@ -54,5 +55,10 @@ public Sighting CreateSighting(CreateSightingRequest request)
 
             return _sightings.CreateSighting(newSighting);
         }
+
+        public Sighting GetSightingById(int sightingId)
+        {
+            return _sightings.GetSightingById(sightingId);
+        }
     }
-}
+}
```

Comment:
Ah sorry Disha, but we always want exactly 1 blank line at the end of every file!

Reply:
Done